### PR TITLE
Remove support for Connect

### PR DESCRIPTION
**Pull request by thornbill**
Removes support for Connect and related strings

### CODE REVIEW

File: app/src/main/java/org/jellyfin/androidtv/util/Utils.java [hunk 1]
```diff
@@ -1635,6 +1628,7 @@ public void onResponse(ArrayList<ServerInfo> serverResponse) {
                     }
                 });
                 break;
```
Comment (JustAMan) [inline app/src/main/java/org/jellyfin/androidtv/util/Utils.java]: How is it possible to get in this state if you're removing connect login?

Comment (thornbill) [reply app/src/main/java/org/jellyfin/androidtv/util/Utils.java]: That state is set by the Java API client when no servers are available. That logic should probably be removed at some point.

https://github.com/jellyfin/jellyfin-apiclient-java/blob/master/android/src/main/java/mediabrowser/apiinteraction/connectionmanager/ConnectionManager.java#L133

**Comment by thornbill** [comment]
I just tested automatic discovery and confirmed it still works as expected.

File: app/src/main/java/org/jellyfin/androidtv/util/Utils.java (hunk 1 above)
Comment (JustAMan) [reply app/src/main/java/org/jellyfin/androidtv/util/Utils.java]: Thanks!

**Pull request merged**